qemu/tests/cfg/uefi_check_descriptor_meta-files.cfg

```diff
@@ -0,0 +1,14 @@
+- uefi_check_descriptor_meta-files:
+    only q35
+    only ovmf
+    no Host_RHEL.m5 Host_RHEL.m6
+    start_vm = no
+    type = uefi_check_descriptor_meta-files
+    query_files = "rpm -ql %s"
+    file_suffix = ".json"
+    Host_RHEL.m7:
+        query_package = "rpm -qa | grep OVMF"
+        ovmf_package_name = "OVMF.*el7.noarch"
```

Comment:
1. is it possible that 'pkg.noarch' might be changed to 'pkg.x86_64', I mean OVMF is the key word we have to check, but the arch might differ across platforms.
2. OVMF\..*el7\..*?

Reply:
Currently, there are not related cases on other platforms. But you are right, I will update it.